### PR TITLE
Alejandro es translations

### PR DESCRIPTION
**Pull request by janorivera**
Primary reviewer:
@GPaoloni  @Humairaa127 
## Description
Hey Gian and Humairaa, I think that this now is pretty complete. I've used as source both the flexUI in english and the one in portuguese.
This PR should only add 2 new files and spanish in the pluginhelper.js

### CODE REVIEW

**Comment by stephenhand** [comment]
@janorivera - As well as mentioning primary reviewers in the description, could you add reviewers to the list on the 'Reviewers' list (on right hand side of the 'Conversation' tab) - this way it will appear in their 'Review Requests' list on their Github account.

Typically we add the primary reviewers, plus the rest of the engineering team - it doesn't mean you want reviews from ALL those people, it just gives them a heads up in case they want to review it

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+    "TaskLineQueue": "Cola: {{task.queueName}}",
+    "Anonymous": "Anónimo",
+    "AnonymousParticipant": "Anónimo",
+    "TaskTabAgentIncomingLabel": "Entrante",
+    "LoadingChatCanvas": "Cargando",
+    "TaskTabHeaderChat": "Charlar",
+    "TaskTabHeaderCall": "Llamar",
+    "TaskHeaderStatusAccepted": "En vivo | {{helper.durationSinceUpdate}}",
+    "TaskHeaderStatusWrapup": "Concluir | {{helper.durationSinceUpdate}}",
+    "CallParticipantStatusLive": "Vivir",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:64]: Way too literal :P

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+    "Bottombar-close": "",
+    "BottomBar-OpenNewCase": "Abrir nuevo caso",
+    "BottomBar-AddToExistingCase": "Agregar al caso existente",
+    "Case-TimelineSection": "Línea de tiempo",
+    "Case-AddHouseholdSection": "Información del hogar",
+    "Case-AddPerpetratorSection": "Perpetradores",
+    "Case-AddIncidentSection": "Incidentes",
+    "Case-CaseSummarySection": "Resumen del caso",
+    "Case-ViewButton": "Vista",
+    "CaseHeader-Opened": "Abrió",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:124]: "Abierto"?

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+    "CaseHeader-Updated": "Actualizado",
+    "CaseHeader-Closed": "Cerrado",
+    "CaseHeader-NoData": "Sin datos",
+    "Case-CaseDetailsDateOpened": "Abrió",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:128]: Ídem

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+    "Case-CaseDetailsLastUpdated": "Última actualización / cerrada",
+    "Case-CaseDetailsFollowUpDate": "Fecha de seguimiento",
+    "Case-CaseDetailsStatusLabel": "Estado",
+    "Case-ChildIsAtRisk": "El niño está en riesgo",
+    "Case-Note": "Nota",
+    "Case-Referral": "Remisión",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:134]: This is the only place where referral is not "referencia".

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+    "BottomBar-SaveAndAddAnotherDocument": "Guardar y subir otro documento",
+    "BottomBar-SaveDocument": "Guardar documento",
+    "Case-AddNote": "Añadir la nota",
+    "BottomBar-SaveNote": "Ahorrar nota",
+    "Case-AddReferral": "Agregar referencia",
+    "BottomBar-SaveReferral": "Guardar referencia",
+    "Case-AddPerpetrator": "Agregar perpetrador",
+    "Case-ViewPerpetrator": "Ver autor",
+    "Case-ViewPerpetratorTitle": "Autor",
+    "BottomBar-SaveAndAddAnotherPerpetrator": "Guarde y agregue otro perpetrador",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:152]: All of this are weird, "Guardar y agregar otro xxxxx" might be better?

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 6]
```diff
@@ -0,0 +1,441 @@
...
+    "Case-EditIncident": "Editar incidente",
+    "Case-EditDocument": "Editar documento",
+    "Case-ViewIncident": "Ver incidente",
+    "Case-ViewIncidentTitle": "Incidente",
+    "BottomBar-SaveIncident": "Salvar el incidente",
+    "BottomBar-Close": "Cerrar",
+    "Case-Contact": "Contacto",
+    "BottomBar-CancelNewCaseAndClose": "Sí, cancele el nuevo caso y cierre",
+    "BottomBar-Update": "Actualizar",
+    "BottomBar-AddThisContactToExistingCase": "Agregue este contacto al caso existente",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:174]: "Agregar ..."?

File: plugin-hrm-form/src/translations/es-ES/flexUI.json [hunk 7]
```diff
@@ -0,0 +1,441 @@
...
+    "CallType-hangup": "Colgar",
+    "CallType-wrongnumber": "Número equivocado",
+    "CallType-abusive": "Abusivo",
+    "CallType-CloseContact": "Contacto cercano",
+    "ConnectDialog-Child": "¿Copiar información del niño de este registro a un nuevo contacto?",
+    "ConnectDialog-Caller": "¿Copiar información de la persona que llama de este registro a un nuevo contacto?",
+    "QueueCard-Name": "Nombre de la cola:",
+    "CaseHeader-ViewCase": "Vista de caso",
+    "Transfer-AcceptTransferButton": "Aceptar transferencia",
+    "Transfer-RejectTransferButton": "Transferencia de rechazo",
```
Comment (GPaoloni) [inline plugin-hrm-form/src/translations/es-ES/flexUI.json:222]: "Rechazar transferencia"?

**Review by GPaoloni — APPROVED** [review]
Looks good! I've left observations about the translations, as some of them sounds weird in my head, but ignore them if this is fine as it is.

**Comment by GPaoloni** [comment]
:rocket:

**Pull request merged**